src/components/alternative-header/stories.tsx

```diff
@@ -34,16 +33,16 @@ const Template: Story<AlternativeHeaderProps> = (props) => {
         onClick: () => setActive('one more tab'),
       },
       // if the storybook page hangs, comment the tab below. There is an unknown memory leak, which breaks the page.
-      {
-        title: 'last tab',
-        isActive: active === 'last tab',
-        altContent: (
-          <Value as="span" lineHeight="13px">
-            38
-          </Value>
-        ),
-        onClick: () => setActive('last tab'),
-      },
+      // {
+      //   title: 'last tab',
+      //   isActive: active === 'last tab',
+      //   altContent: (
```

Comment:
I don't have a slightest clue why this thing breaks storybooks all the time, but it does